content/en/tracing/trace_collection/library_config/php.md

```diff
@@ -435,4 +440,4 @@ When the application runs in a docker container, the path `/proc/self` should al
 [7]: https://github.com/openzipkin/b3-propagation
 [8]: https://github.com/openzipkin/b3-propagation#single-header
 [9]: https://www.php.net/manual/en/ini.core.php#ini.open-basedir
-[13]: /agent/guide/network/#configure-ports
\ No newline at end of file
+[13]: /agent/guide/network/#configure-ports
```

Comment:
... what's the diff here? I cannot tell.

Reply:
CLI shows this:
```diff
- [13]: /agent/guide/network/#configure-ports
- \ No newline at end of file
+ [13]: /agent/guide/network/#configure-ports
```
There was no newline char at the end of the file and my vim is configured to add those